pubspec.yaml

```diff
@@ -13,7 +13,7 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  permission_handler: ^4.0.0
+  permission_handler: ^5.0.1+1
```

Comment:
This update would require changes to background_location.dart. More work to follow.